src/admin/js/pages/plugins/filepond/_validations.js

```diff
@@ -0,0 +1,52 @@
+import { createAlert, removeAlert } from './_alerts';
+import { FILE_LOCAL, FILE_PROCESSING_COMPLETE } from './_config';
+
+export function validateMinFilesInFilepond(pond, inputId, minFiles) {
+  if (!minFiles) return true;
+
+  const hasMinFiles = pond.getFiles().length >= minFiles;
+
+  if (!hasMinFiles) {
+    createAlert(
+      inputId,
+      `El mínimo de archivos necesarios es de ${minFiles}`,
+      'primary',
+      false,
+      false,
+      'minFile'
+    );
+  } else {
+    removeAlert(inputId, 'minFile');
+  }
+
+  return hasMinFiles;
+}
+
+export function validateMaxFilesInFilepond(pond, inputId, files = []) {
+  if (!pond.maxFiles) return;
+  const numPondFiles = pond.getFiles().length;
+  const payloadLength = files.length;
+  const exceedFiles = payloadLength + numPondFiles > pond.maxFiles;
+  const hasMaxFiles = numPondFiles === pond.maxFiles;
+  if (exceedFiles || hasMaxFiles) {
+    removeAlert(inputId, 'maxFiles');
+    createAlert(
+      inputId,
+      hasMaxFiles
+        ? `Ha alcanzado el numero máximo de archivos permitidos. Por favor elimine uno si desea agregar otro`
+        : `Ha intentado ingresar más archivos de los permitidos. Por favor, intente con una menor cantidad de archivos`,
```

Comment:
Here I see you have access to the max files and current file number so the error message could be more complete like this, you can rephrase this but the important thing is to give the user the information about the permitted files limits since we have it there:
```suggestion
        : `Ha intentado ingresar más archivos de los permitidos. El máximo de archivos permitidos es de ${pond.maxFiles}, puede ingresar hasta ${pond.maxFiles - numPondFiles} archivos más`,
```

Reply:
Thanks, it is a better message